packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -503,22 +508,40 @@ class PlayBar extends Component {
 const makeMapStateToProps = () => {
   const getCurrentQueueItem = makeGetCurrent()
 
-  const mapStateToProps = (state, props) => ({
-    accountUser: getAccountUser(state),
-    currentQueueItem: getCurrentQueueItem(state),
-    playCounter: getCounter(state),
-    collectible: getCollectible(state),
-    isPlaying: getPlaying(state),
-    isBuffering: getBuffering(state),
-    playingUid: getPlayingUid(state),
-    playbackRate: getPlaybackRate(state),
-    lineupHasTracks: getLineupHasTracks(
+  const mapStateToProps = (state) => {
+    const premiumTrackSignatureMap = getPremiumTrackSignatureMap(state)
+    const lineupEntries = getLineupEntries(
       getLineupSelectorForRoute(state),
       state
-    ),
-    userId: getUserId(state),
-    theme: getTheme(state)
-  })
+    )
+    const lineupHasAccessibleTracks = lineupEntries.some((entry) => {
```

Comment:
Minor nit: If it feels like there are any subtle behaviors in some of the logic here and elsewhere, it might be a good idea to add a little documentation about what's happening and why.